LookupAnything/DataParser.cs

```diff
@@ -445,30 +445,26 @@ private Item CreateRecipeItem(string? inputID, string outputID, MachineRecipeOut
             {
                 if (inputID != null)
                 {
-                    switch (outputID)
+                    switch (ItemRegistry.GetDataOrErrorItem(outputID).ItemId)
                     {
-                        case "(O)342":
                         case "342":
```

Comment:
We should use the qualified item ID instead, to avoid conflicts with other `SObject`-type item IDs (e.g. furniture).

In that case we can also change `ItemRegistry.GetDataOrErrorItem(outputID).ItemId` to `ItemRegistry.QualifyItemId(outputId)`.

Reply:
Done